### DISCUSSION TITLE
Bigger Icons for Download

### DISCUSSION THREAD

**Discussion by AnnieMiteva** [lucide-icons/lucide#1556] [Ideas]
Hey there,

Is it possible to make it so the icons can be downloaded bigger than 48px in PNG?

**Comment by mikeott** [comment]
@AnnieMiteva 

Ideally use SVG instead. 

But if you can't you can 'cheat' but inspecting the size slider handle and changing the **max** value to whatever you want. Example: from this...

`<input id="size" type="range" name="size" value="24" min="16" max="48" step="4">`

...to this...

`<input id="size" type="range" name="size" value="24" min="16" max="512" step="4">`

It makes the page layout go a little crazy but the icons will now download at the new max size. It's not a real solution obviously but it might get you to where you need to go.

**Comment by ericfennis** [comment]
Can you explain what your usecase is?

**Reply by AnnieMiteva** [reply]
We use Lucide in our company as an icon pack for the brand materials and the presentations. Unfortunately, Google Slides does not allow svg and the 48px icons are too small. This is why I am wondering if it is possible to make it so users are able to write the size of the icon or make the slider go above 48px.

**Reply by mikeott** [reply]
SVG is a vector format, the dimensions don't really matter. This means you can bring them into an image editor (Photoshop for example or a free app that lets you open SVG files) and specify any size you want, and then save as a PNG, JPG or whatever.

**Reply by AnnieMiteva** [reply]
I understand. I am a designer and work quite often with SVGs. For me this does not cause an issue. However, there are users that don't have access to Photoshop and might not know how to convert an SVG to PNG online. 

Overall, for designers and developers it is quite easy to use the library, so my request here was for easing the process of using Lucide as PNGs for the rest of the users.

**Reply by chessurisme** [reply]
Did you find any solution, @AnnieMiteva?

**Reply by jamiemlaw** [reply]
Probably one of the best online image converters is https://squoosh.app/. If you drop the SVG version of an icon in there, it will let you resize it and convert it to PNG.

**EDIT:**

Came across this tool that someone built, because they had literally this very issue: https://quickpic.t3.gg/svg-to-png.

[YouTube: This tool annoyed me (so I built a free version)](https://www.youtube.com/watch?v=G2_D2bYFjY4)
